sdk/lib/cmlib/hivewrt.c

```diff
@@ -104,12 +110,13 @@ HvpWriteLog(
         FileOffset += HBLOCK_SIZE;
     }
 
-    Success = RegistryHive->FileSetSize(RegistryHive, HFILE_TYPE_LOG, FileOffset, FileOffset);
+    /* TODO: Check this function. In some cases log file can be corrrupted */
+    /* Success = RegistryHive->FileSetSize(RegistryHive, HFILE_TYPE_LOG, FileOffset, FileOffset);
```

Comment:
It's preferrable to use `#if 0 ... #endif` for commenting out large blocks of code btw.